cgmanifest.json

```diff
@@ -105,8 +105,8 @@
         "type": "other",
         "other": {
           "name": "advancecomp",
-          "version": "2.5",
-          "downloadUrl": "https://github.com/amadvance/advancecomp/archive/v2.5/advancecomp-2.5.tar.gz"
+          "version": "2.6",
+          "downloadUrl": "https://github.com/amadvance/advancecomp/archive/refs/tags/v2.6.tar.gz"
```

Comment:
Upload to blobstore: [link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=657008&view=results)